packages/ui/src/working-groups/hooks/useMyEarnings.ts

```diff
@@ -0,0 +1,40 @@
+import BN from 'bn.js'
+import { startOfMonth, subHours } from 'date-fns'
+
+import { useAccounts } from '@/accounts/hooks/useAccounts'
+import { RewardPaidEventWhereInput } from '@/common/api/queries'
+
+import { RewardPaidEventFieldsFragment, useGetRewardsQuery } from '../queries'
+
+export interface UseMyEarnings {
+  last24hours: BN | null
+  month: BN | null
+}
+
+export function useMyEarnings(): UseMyEarnings {
+  const { allAccounts } = useAccounts()
+
+  const where: RewardPaidEventWhereInput = {
+    rewardAccount_in: allAccounts.map((account) => account.address),
```

Comment:
A reward account can be changed during the last 30 day period. We need to check all payouts related to the user.

Reply:
Ooops, forgot about this one